src/pay/batch_payer.py

```diff
@@ -261,42 +281,50 @@ def pay(self, payment_items_in, dry_run=None):
         if payment_address_balance is not None:
 
             logger.info(
-                "Current balance in payout address is {:,} mutez.".format(
-                    payment_address_balance
+                "Current balance in payout address is {:,} tez.".format(
+                    payment_address_balance / MUTEZ
                 )
             )
 
             number_future_payable_cycles = (
-                int(payment_address_balance / total_amount_to_pay) - 1
+                int(
```

Comment:
This is horrible formatting. Why can't this all be on 1 line? Line 294 is literally "-1" What does that even mean? Don't line wrap if it's not necessary.

Reply:
That comes from the `black . ` formatting that we use. It is really a joke here....

Reply:
#563